### PR TITLE
ddl: reduce the interval for checking "create table/schema"

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve? 
After we put the DDL job into the job queue, the owner will handle the job in the background.  We check every "tick" if the current job is completed. Handling the operation of "create table/schema" will be faster.

### What is changed and how it works?
I changed the interval of "create table/schema"  to 500ms.

### CODE REVIEW

File: ddl/ddl.go [hunk 1]
```diff
@@ -447,6 +447,9 @@ func checkJobMaxInterval(job *model.Job) time.Duration {
 	if job.Type == model.ActionAddIndex {
 		return 3 * time.Second
 	}
+	if job.Type == model.ActionCreateTable {
```
Comment (winkyao) [inline ddl/ddl.go]: Why only handle model.ActionCreateTable？

Comment (winkyao) [inline ddl/ddl.go]: Please add a `TODO` here.

Comment (shenli) [reply ddl/ddl.go]: Why not consider CreateDatabase/DropTable....?

Comment (zimulala) [reply ddl/ddl.go]: @shenli 
"DropTable" has 4 states. So this operation takes longer than it takes.

Comment (zhexuany) [reply ddl/ddl.go]: I am confused by the context. Anyone help me out? 

> request add a TODO.
TODO for what? 
> Why not consider CreateDatabase/DropTable....? 
you mean we reduce tick of `CreateDatabase/DropTable` to 500ms?

Comment (shenli) [reply ddl/ddl.go]: Yes, they are the same story with CreateTable.
> you mean we reduce tick of CreateDatabase/DropTable to 500ms?

Comment (zimulala) [reply ddl/ddl.go]: @shenli 
This is the estimated time.
```
tidb> create database db;
Query OK, 0 rows affected (0.16 sec)
tidb> create table db.x(a int, b int);
Query OK, 0 rows affected (0.19 sec)
tidb> drop table db.x;
Query OK, 0 rows affected (0.39 sec)
tidb> drop database db;
Query OK, 0 rows affected (0.35 sec)
```

**Pull request merged**